itou/www/apply/views/submit_views.py

```diff
@@ -56,6 +56,9 @@ def start(request, siae_pk):
     if request.user.is_siae_staff and not siae.has_member(request.user):
         raise PermissionDenied(_("Vous ne pouvez postuler pour un candidat que dans votre structure."))
 
+    if siae.block_job_applications:
+        raise Http404(_("Cette organisation n'accepte plus de candidatures pour le moment."))
```

Comment:
En fait il faut bloquer seulement si l'émetteur n'est pas la SIAE elle-même, sinon elle ne pourra pas obtenir de PASS IAE par elle-même.

Reply:
Cool ! Juste ajouter un commentaire car le sens ne saute pas aux yeux en lisant le code ?

`# Refuse all applications except those issued by the SIAE`